src/foundations/focus.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Focus
+summary: Indicating and managing focus is integral to keyboard accessibility
+version: 0.1.0
+published: false
+---
+
+## Introduction
+
+When discussing access to web content, we often use the term assistive technologies[^1]: software and devices to help those with disabilities. However, assistive technologies, unlike _adaptive technologies_ (such as hearing aids), do not just address disability. A screen reader is not just for blind and vision-impaired users; it can be called upon to address educational and productivity requirements as well.
+
+The keyboard, whether mechanical or virtual, is used by most anyone at some point during interaction. The term _"keyboard users"_, used to demarcate a disability demographic, is therefore misleading. All people use keyboards; some depend on them more than others. 
```

Comment:
Actually, correcting myself now, BBC editorial rightly says not to use “the disabled.” So that would become: “It’s misleading to think that the term "keyboard users" only applies to disabled people.”